.gitignore

```diff
@@ -3,4 +3,5 @@ imgs/
 target
 latest.log
 *.json
-*.iml
\ No newline at end of file
+*.iml
+tags
```

Comment:
Why?

Reply:
ctags automation in many setup makes this file automatically whenever the editor opens, and you don't need a bunch of tagfiles randomly do you?